charts/library/common/Chart.yaml

```diff
@@ -14,4 +14,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |-
     - kind: changed
-      description: Support services have extraSelectorLabels
+      description: Added support for Gateway API Routes
```

Comment:
Please don't remove the "old" changelog for the `development` branch, but instead add to it.

Reply:
If I figured correctly the current HEAD is released and everything is "new" so I ended up clearing the changes again, let me know if I misunderstood the situation.